src/components/Layout/Desktop.jsx

```diff
@@ -1,26 +1,31 @@
-import React, { useState } from 'react';
-import { Map, Figures, DataElement, Contributors } from '..';
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Map, Figures, DataElement, Contributors } from "..";
 
-import { Layer } from 'baseui/layer';
-import { Button, KIND, SIZE } from 'baseui/button';
-import { Block } from 'baseui/block';
-import { Modal, ModalHeader, ModalBody, ROLE } from 'baseui/modal';
-import { Paragraph3, Label2 } from 'baseui/typography';
+import { Layer } from "baseui/layer";
+import { Button, KIND, SIZE } from "baseui/button";
+import { Block } from "baseui/block";
+import { Modal, ModalHeader, ModalBody, ROLE } from "baseui/modal";
+import { Paragraph3, Label2 } from "baseui/typography";
 
-import { useTheme } from '../../contexts/ThemeContext';
-import { StyledLink } from 'baseui/link';
-import DailyGrowth from '../DailyGrowth/DailyGrowth';
+import { switchLanguage } from "../../helpers/switchLanguage";
+
+import { useTheme } from "../../contexts/ThemeContext";
+import { StyledLink } from "baseui/link";
+import DailyGrowth from "../DailyGrowth/DailyGrowth";
 
 export default function Layout() {
+  const { t, i18n } = useTranslation();
   const [isOpen, setIsOpen] = useState(false);
   const { useDarkTheme, setUseDarkTheme } = useTheme();
 
   return (
     <>
       <Layer>
-        <Map className={useDarkTheme ? 'dark-theme' : ''} />
+        <Map className={useDarkTheme ? "dark-theme" : ""} />
       </Layer>
       <Layer>
+<<<<<<< HEAD
```

Comment:
I think you left unresolved confict here.

Reply:
Thanks, I had a couple of them, so didn't notice this one. Fixed it.